.gitignore

```diff
@@ -1,6 +1,4 @@
 /.nyc_output
-/bin
 /coverage
-/lib
```

Comment:
Please revert this change.
The result of builds should not be included in git repo.